web/source/kmwbase.ts

```diff
@@ -183,8 +183,8 @@ namespace com.keyman {
      */
     ['shutdown']() {
       // Disable page focus/blur events, which can sometimes trigger and cause parallel KMW instances in testing.
-      this.util.detachDOMEvent(window, 'focus', this['pageFocusHandler'], false);
-      this.util.detachDOMEvent(window, 'blur', this['pageFocusHandler'], false);
+      this.util.detachDOMEvent(window, 'focus', (this as any).pageFocusHandler, false);
+      this.util.detachDOMEvent(window, 'blur', (this as any).pageFocusHandler, false);
```

Comment:
Huh, why do we need the typecast here?

Reply:
Because in keymanweb.ts, we aren't (formally) part of the class definition when we define the method.

It was either this or relocate the method to kmwbase.ts, and I figured relocating would be messier.

Reply:
OK. It'll probably break aggressive minification though!

Reply:
The main Web engine's build only uses "simple" optimizations, not "advanced" (i.e. agressive) ones.  We're only "aggressive" for a couple of the UI modules.

Though, I could do that relocation if you'd prefer.

Reply:
Let's go ahead and relocate it. I know we don't use advanced optimizations at present but it'd be good not to further entrench that!

Reply:
And I just double-checked, to be sure; the minified 'sample' page functions fine with this change.